components/Workspaces/TrackedRepoWizard/TrackedRepoWizard.tsx

```diff
@@ -24,6 +24,18 @@ type TrackedReposStep =
   | "filterPastedRepos"
   | "pickOrgRepos";
 
+async function getCaseSensitiveOrgName(orgSearchTerm: string) {
+  const response = await fetch(`https://api.github.com/orgs/${orgSearchTerm}`);
+
+  if (response.status === 200) {
+    // The organization name is case sensitive even if our search was case insensitive.
+    // Return the case sensitive organization name so we can use it to get the organization's repositories.
+    const { login } = await response.json();
```

Comment:
This is very surprising to me especially given how our API handles all logins / orgs / repo names as case insensitive. Any reason? Is it that the org "Name" is different from the actual org login (that we use in the API). I.e., `https://github.com/citi` is the same as `https://github.com/CITI` but the "name" is `Citi`?

Reply:
Yeah, this surprised me as well. It’s not in our API though. It’s a React hook that hits the GitHub API.

See https://github.com/open-sauced/app/blob/beta/lib/hooks/useGetOrgRepos.ts

Reply:
It was a casing mismatch on the client-side when filtering client-side. All sorted out in 290a778b1e6ea3ccccf78c377e538ea3d676510b

Reply:
Solid!! 👏🏼 👏🏼